docs/modules/deployment/pages/air-gap/air-gap-install.adoc

```diff
@@ -0,0 +1,151 @@
+
+### Step 1: Create a Temporary Package Directory
+
+Create a base directory to organize all downloaded packages.  This directory will become the base directory and dependencies will be included inside sub-folders to aid in knowing what order to install them.
+
+```bash
+mkdir ./install
+```
+
+### Step 2: Install OpenNMS Repository
+
+Add the {page-component-title} repository and import the GPG key:
+
+ifeval::["{page-component-title}" == "Horizon"]
+```bash
+dnf -y install https://yum.opennms.org/repofiles/opennms-repo-stable-rhel9.noarch.rpm
+rpm --import https://yum.opennms.org/OPENNMS-GPG-KEY
+```
+endif::[]
+
+ifeval::["{page-component-title}" == "Meridian"]
+```bash
+cat << EOF | sudo tee /etc/yum.repos.d/opennms-meridian.repo
+[meridian]
+name=Meridian for Red Hat Enterprise Linux and CentOS
+baseurl=https://REPO_USER:REPO_PASS@meridian.opennms.com/packages/2024/stable/rhel9
+gpgcheck=1
+gpgkey=http://yum.opennms.org/OPENNMS-GPG-KEY
+EOF
+
+sudo rpm --import https://yum.opennms.org/OPENNMS-GPG-KEY
+```
+endif::[]
+
+
+### Step 3: Download OpenJDK
+
+Download and install OpenJDK packages (OpenJDK 17 here but please refer to system requirements):
+
+```bash
+dnf install --downloadonly --downloaddir=/tmp/opennms_rpms java-17-openjdk-devel
+mv ./tmp/opennms_rpms ./install/01-openjdk
+dnf install -y java-17-openjdk-devel
+```
+
+### Step 4: Download PostgreSQL
+
+Download PostgreSQL server packages (please refer to system requirements for version support):
+
+```bash
+dnf install --downloadonly --downloaddir=./rpmtmp postgresql-server
+mv ./tmp/opennms_rpms ./install/02-postgresql
+dnf install -y postgresql-server
+```
+
+### Step 5: Download OpenNMS Core Packages
+
+Download the basic OpenNMS packages. If you need other packages, this would be the step to add them in.
+
+ifeval::["{page-component-title}" == "Horizon"]
+```bash
+dnf install --downloadonly --downloaddir=./rpmtmp opennms-core opennms-webapp-jetty
+mv ./tmp/opennms_rpms ./install/04-opennms
+dnf install -y opennms-core opennms-webapp-jetty
+```
+endif::[]
+
+ifeval::["{page-component-title}" == "Meridian"]
+```bash
+dnf install --downloadonly --downloaddir=./rpmtmp meridian-core meridian-webapp-jetty
+mv ./tmp/opennms_rpms ./install/04-opennms
+dnf install -y meridian-core meridian-webapp-jetty
+```
+endif::[]
+
+## Part 2: Installation on Air-Gapped System
+
+Transfer the files to your air-gapped system via USB, DVD, or link:https://www.rfc-editor.org/rfc/rfc2549.html[secure file transfer].
+
+### Step 1: Locate The Files
+
+Locate the directory where the install files are located after you have transfered them.
+```bash
+cd /mnt/opennms_transfer/install
+```
+
+### Step 2: Install OpenJDK
+
+```bash
+cd /mnt/01-openjdk
+rpm -ivh <directory>/*.rpm
+```
+
+### Step 3: Install PostgreSQL
+
+```bash
+cd /mnt/02-postgresql
+rpm -ivh <directory>/*.rpm
+```
+
+### Step 3: Install OpenNMS Core
+
+```bash
+cd /mnt/04-opennms
+rpm -ivh <directory>/*.rpm
+```
```

Comment:
Separating everything into its own directory seems unnecessarily complicated.  It can all go one place, and you can `yum localinstall /that/place/*.rpm`